app/views/discovered_hosts/_discovered_hosts_list.html.erb

```diff
@@ -38,8 +38,39 @@
       <td class="hidden-tablet hidden-xs"><%= host.primary_interface.try(:subnet) %></td>
       <td class="hidden-tablet hidden-xs"><%= disc_report_column(host) %></td>
       <td>
+        <!-- Modal -->
```

Comment:
>can you create just one instance of it and little bit of javascript that sets required things?

It's doable, but why? It adds more complexity to the view, but I can't see a real benefit except maybe smaller HTML that gets to the client.
From programming point of view, there is only one place to change if we need changes to the modal.
If the amount of the HTML is an issue, we can offload it completely to AJAX.